dask/bag/tests/test_bag.py

```diff
@@ -366,6 +366,9 @@ def test_foldby():
     assert set(c) == set(reduceby(iseven, lambda acc, x: acc + x, L, 0).items())
     assert c.name == b.foldby(iseven, add, 0, add, 0).name
 
+    d = b.foldby(iseven, add, 0, add, 0, split_every=2)
+    assert c.compute() == d.compute()
```

Comment:
It would be useful to verify that this works with a few different values of split_every.  In particular we might want to test behavior of the special values `None` and `False` as well as a few numeric values that both divide and don't divide the number of partitions evenly.  This might warrant its own testing function.

Reply:
Will do.

Reply:
Done.